bulk-model-sync-lib/src/jvmMain/kotlin/org/modelix/model/sync/bulk/PlatformSpecific.kt

```diff
@@ -35,6 +35,12 @@ fun ModelImporter.import(jsonFile: File) {
     import(data)
 }
 
+fun ModelImporter.importFilesAsRootChildren(files: Collection<File>) {
+    val models = files.map { ModelData.fromJson(it.readText()) }
+    import(mergeModelData(*models.toTypedArray()))
```

Comment:
Now we got the same problem one level down, right?

Reply:
btw, if the array changes are solely related to performance, the angular conventions, to which conventional-commits refer for further commit types, also contain a `perf` commit type: https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#type